2024-01-07 03:26:14.025 | ERROR    | utils.loggings:exception:40 - 400 Bad Request: The browser (or proxy) sent a request that this server could not understand.
Traceback (most recent call last):

  File "D:\Python\Python39\lib\threading.py", line 937, in _bootstrap
    self._bootstrap_inner()
    │    └ <function Thread._bootstrap_inner at 0x00000265FFC74670>
    └ <Thread(Thread-1, started daemon 30420)>

  File "D:\Python\Python39\lib\threading.py", line 980, in _bootstrap_inner
    self.run()
    │    └ <function Thread.run at 0x00000265FFC743A0>
    └ <Thread(Thread-1, started daemon 30420)>

  File "D:\Python\Python39\lib\threading.py", line 917, in run
    self._target(*self._args, **self._kwargs)
    │    │        │    │        │    └ {}
    │    │        │    │        └ <Thread(Thread-1, started daemon 30420)>
    │    │        │    └ ()
    │    │        └ <Thread(Thread-1, started daemon 30420)>
    │    └ <bound method BaseWSGIServer.serve_forever of <werkzeug.serving.BaseWSGIServer object at 0x00000265A36D8FD0>>
    └ <Thread(Thread-1, started daemon 30420)>

  File "D:\Python\Python39\lib\site-packages\werkzeug\serving.py", line 806, in serve_forever
    super().serve_forever(poll_interval=poll_interval)
                                        └ 0.5

  File "D:\Python\Python39\lib\socketserver.py", line 237, in serve_forever
    self._handle_request_noblock()
    │    └ <function BaseServer._handle_request_noblock at 0x00000265A0BB7AF0>
    └ <werkzeug.serving.BaseWSGIServer object at 0x00000265A36D8FD0>

  File "D:\Python\Python39\lib\socketserver.py", line 316, in _handle_request_noblock
    self.process_request(request, client_address)
    │    │               │        └ ('127.0.0.1', 51355)
    │    │               └ <socket.socket fd=1144, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 5000), raddr=...
    │    └ <function BaseServer.process_request at 0x00000265A0BB7CA0>
    └ <werkzeug.serving.BaseWSGIServer object at 0x00000265A36D8FD0>

  File "D:\Python\Python39\lib\socketserver.py", line 347, in process_request
    self.finish_request(request, client_address)
    │    │              │        └ ('127.0.0.1', 51355)
    │    │              └ <socket.socket fd=1144, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 5000), raddr=...
    │    └ <function BaseServer.finish_request at 0x00000265A0BB7DC0>
    └ <werkzeug.serving.BaseWSGIServer object at 0x00000265A36D8FD0>

  File "D:\Python\Python39\lib\socketserver.py", line 360, in finish_request
    self.RequestHandlerClass(request, client_address, self)
    │    │                   │        │               └ <werkzeug.serving.BaseWSGIServer object at 0x00000265A36D8FD0>
    │    │                   │        └ ('127.0.0.1', 51355)
    │    │                   └ <socket.socket fd=1144, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 5000), raddr=...
    │    └ <class 'werkzeug.serving.WSGIRequestHandler'>
    └ <werkzeug.serving.BaseWSGIServer object at 0x00000265A36D8FD0>

  File "D:\Python\Python39\lib\socketserver.py", line 747, in __init__
    self.handle()
    │    └ <function WSGIRequestHandler.handle at 0x00000265A0DF2550>
    └ <werkzeug.serving.WSGIRequestHandler object at 0x00000265A379F6A0>

  File "D:\Python\Python39\lib\site-packages\werkzeug\serving.py", line 390, in handle
    super().handle()

  File "D:\Python\Python39\lib\http\server.py", line 425, in handle
    self.handle_one_request()
    │    └ <function BaseHTTPRequestHandler.handle_one_request at 0x00000265A0CE2160>
    └ <werkzeug.serving.WSGIRequestHandler object at 0x00000265A379F6A0>

  File "D:\Python\Python39\lib\http\server.py", line 413, in handle_one_request
    method()
    └ <bound method WSGIRequestHandler.run_wsgi of <werkzeug.serving.WSGIRequestHandler object at 0x00000265A379F6A0>>

  File "D:\Python\Python39\lib\site-packages\werkzeug\serving.py", line 362, in run_wsgi
    execute(self.server.app)
    │       │    │      └ <werkzeug.debug.DebuggedApplication object at 0x00000265A36D86A0>
    │       │    └ <werkzeug.serving.BaseWSGIServer object at 0x00000265A36D8FD0>
    │       └ <werkzeug.serving.WSGIRequestHandler object at 0x00000265A379F6A0>
    └ <function WSGIRequestHandler.run_wsgi.<locals>.execute at 0x00000265A38B5160>

  File "D:\Python\Python39\lib\site-packages\werkzeug\serving.py", line 325, in execute
    for data in application_iter:
                └ <generator object DebuggedApplication.debug_application at 0x00000265A3799430>

  File "D:\Python\Python39\lib\site-packages\werkzeug\debug\__init__.py", line 330, in debug_application
    app_iter = self.app(environ, start_response)
               │    │   │        └ <function WSGIRequestHandler.run_wsgi.<locals>.start_response at 0x00000265A374F670>
               │    │   └ {'wsgi.version': (1, 0), 'wsgi.url_scheme': 'http', 'wsgi.input': <_io.BufferedReader name=1144>, 'wsgi.errors': <_io.TextIOW...
               │    └ <Flask 'app'>
               └ <werkzeug.debug.DebuggedApplication object at 0x00000265A36D86A0>

  File "D:\Python\Python39\lib\site-packages\flask\app.py", line 2213, in __call__
    return self.wsgi_app(environ, start_response)
           │    │        │        └ <function WSGIRequestHandler.run_wsgi.<locals>.start_response at 0x00000265A374F670>
           │    │        └ {'wsgi.version': (1, 0), 'wsgi.url_scheme': 'http', 'wsgi.input': <_io.BufferedReader name=1144>, 'wsgi.errors': <_io.TextIOW...
           │    └ <function Flask.wsgi_app at 0x00000265A131B790>
           └ <Flask 'app'>

  File "D:\Python\Python39\lib\site-packages\flask\app.py", line 2190, in wsgi_app
    response = self.full_dispatch_request()
               │    └ <function Flask.full_dispatch_request at 0x00000265A1318CA0>
               └ <Flask 'app'>

  File "D:\Python\Python39\lib\site-packages\flask\app.py", line 1484, in full_dispatch_request
    rv = self.dispatch_request()
         │    └ <function Flask.dispatch_request at 0x00000265A1318C10>
         └ <Flask 'app'>

  File "D:\Python\Python39\lib\site-packages\flask\app.py", line 1469, in dispatch_request
    return self.ensure_sync(self.view_functions[rule.endpoint])(**view_args)
           │    │           │    │              │    │            └ {}
           │    │           │    │              │    └ 'orders.queryUser'
           │    │           │    │              └ <Rule '/orders/queryUser' (OPTIONS, GET, HEAD) -> orders.queryUser>
           │    │           │    └ {'static': <function Flask.__init__.<locals>.<lambda> at 0x00000265A1F588B0>, 'apiversion.Apiversion': <function View.as_view...
           │    │           └ <Flask 'app'>
           │    └ <function Flask.ensure_sync at 0x00000265A1318EE0>
           └ <Flask 'app'>

  File "E:\code\backend\api\api_1_0\ordersResource\urls.py", line 27, in queryUser
    return  OrdersOtherResource.queryUser()
            │                   └ <classmethod object at 0x00000265A35E73D0>
            └ <class 'api_1_0.ordersResource.ordersOtherResource.OrdersOtherResource'>

> File "E:\code\backend\api\api_1_0\ordersResource\ordersOtherResource.py", line 68, in queryUser
    kwargs = parser.parse_args()
             │      └ <function RequestParser.parse_args at 0x00000265A31784C0>
             └ <flask_restful.reqparse.RequestParser object at 0x00000265A38C8EB0>

  File "D:\Python\Python39\lib\site-packages\flask_restful\reqparse.py", line 328, in parse_args
    value, found = arg.parse(req, self.bundle_errors)
                   │   │     │    │    └ False
                   │   │     │    └ <flask_restful.reqparse.RequestParser object at 0x00000265A38C8EB0>
                   │   │     └ <Request 'http://127.0.0.1:5000/orders/queryUser' [GET]>
                   │   └ <function Argument.parse at 0x00000265A3178310>
                   └ Argument('UserID', default=None, dest=None, required=True, ignore=False, location=form, type="<class 'str'>", choices=(), act...

  File "D:\Python\Python39\lib\site-packages\flask_restful\reqparse.py", line 246, in parse
    self.handle_validation_error(ValueError(error_msg), bundle_errors)
    │    │                                  │           └ False
    │    │                                  └ 'Missing required parameter in the post body'
    │    └ <function Argument.handle_validation_error at 0x00000265A3178280>
    └ Argument('UserID', default=None, dest=None, required=True, ignore=False, location=form, type="<class 'str'>", choices=(), act...

  File "D:\Python\Python39\lib\site-packages\flask_restful\reqparse.py", line 173, in handle_validation_error
    flask_restful.abort(400, message=msg)
    │             │                  └ {'UserID': '用户ID数据类型不匹配'}
    │             └ <function abort at 0x00000265A312BD30>
    └ <module 'flask_restful' from 'D:\\Python\\Python39\\lib\\site-packages\\flask_restful\\__init__.py'>

  File "D:\Python\Python39\lib\site-packages\flask_restful\__init__.py", line 32, in abort
    original_flask_abort(http_status_code)
    │                    └ 400
    └ <function abort at 0x00000265A117C670>

  File "D:\Python\Python39\lib\site-packages\flask\helpers.py", line 277, in abort
    current_app.aborter(code, *args, **kwargs)
    │                   │      │       └ {}
    │                   │      └ ()
    │                   └ 400
    └ <Flask 'app'>

  File "D:\Python\Python39\lib\site-packages\werkzeug\exceptions.py", line 861, in __call__
    raise self.mapping[code](*args, **kwargs)
          │    │       │      │       └ {}
          │    │       │      └ ()
          │    │       └ 400
          │    └ {400: <class 'werkzeug.exceptions.BadRequest'>, 401: <class 'werkzeug.exceptions.Unauthorized'>, 403: <class 'werkzeug.except...
          └ <werkzeug.exceptions.Aborter object at 0x00000265A0B63970>

werkzeug.exceptions.BadRequest: 400 Bad Request: The browser (or proxy) sent a request that this server could not understand.
2024-01-07 03:26:17.934 | ERROR    | utils.loggings:exception:40 - 400 Bad Request: The browser (or proxy) sent a request that this server could not understand.
Traceback (most recent call last):

  File "D:\Python\Python39\lib\threading.py", line 937, in _bootstrap
    self._bootstrap_inner()
    │    └ <function Thread._bootstrap_inner at 0x00000265FFC74670>
    └ <Thread(Thread-1, started daemon 30420)>

  File "D:\Python\Python39\lib\threading.py", line 980, in _bootstrap_inner
    self.run()
    │    └ <function Thread.run at 0x00000265FFC743A0>
    └ <Thread(Thread-1, started daemon 30420)>

  File "D:\Python\Python39\lib\threading.py", line 917, in run
    self._target(*self._args, **self._kwargs)
    │    │        │    │        │    └ {}
    │    │        │    │        └ <Thread(Thread-1, started daemon 30420)>
    │    │        │    └ ()
    │    │        └ <Thread(Thread-1, started daemon 30420)>
    │    └ <bound method BaseWSGIServer.serve_forever of <werkzeug.serving.BaseWSGIServer object at 0x00000265A36D8FD0>>
    └ <Thread(Thread-1, started daemon 30420)>

  File "D:\Python\Python39\lib\site-packages\werkzeug\serving.py", line 806, in serve_forever
    super().serve_forever(poll_interval=poll_interval)
                                        └ 0.5

  File "D:\Python\Python39\lib\socketserver.py", line 237, in serve_forever
    self._handle_request_noblock()
    │    └ <function BaseServer._handle_request_noblock at 0x00000265A0BB7AF0>
    └ <werkzeug.serving.BaseWSGIServer object at 0x00000265A36D8FD0>

  File "D:\Python\Python39\lib\socketserver.py", line 316, in _handle_request_noblock
    self.process_request(request, client_address)
    │    │               │        └ ('127.0.0.1', 51361)
    │    │               └ <socket.socket fd=1144, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 5000), raddr=...
    │    └ <function BaseServer.process_request at 0x00000265A0BB7CA0>
    └ <werkzeug.serving.BaseWSGIServer object at 0x00000265A36D8FD0>

  File "D:\Python\Python39\lib\socketserver.py", line 347, in process_request
    self.finish_request(request, client_address)
    │    │              │        └ ('127.0.0.1', 51361)
    │    │              └ <socket.socket fd=1144, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 5000), raddr=...
    │    └ <function BaseServer.finish_request at 0x00000265A0BB7DC0>
    └ <werkzeug.serving.BaseWSGIServer object at 0x00000265A36D8FD0>

  File "D:\Python\Python39\lib\socketserver.py", line 360, in finish_request
    self.RequestHandlerClass(request, client_address, self)
    │    │                   │        │               └ <werkzeug.serving.BaseWSGIServer object at 0x00000265A36D8FD0>
    │    │                   │        └ ('127.0.0.1', 51361)
    │    │                   └ <socket.socket fd=1144, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 5000), raddr=...
    │    └ <class 'werkzeug.serving.WSGIRequestHandler'>
    └ <werkzeug.serving.BaseWSGIServer object at 0x00000265A36D8FD0>

  File "D:\Python\Python39\lib\socketserver.py", line 747, in __init__
    self.handle()
    │    └ <function WSGIRequestHandler.handle at 0x00000265A0DF2550>
    └ <werkzeug.serving.WSGIRequestHandler object at 0x00000265A38C8A60>

  File "D:\Python\Python39\lib\site-packages\werkzeug\serving.py", line 390, in handle
    super().handle()

  File "D:\Python\Python39\lib\http\server.py", line 425, in handle
    self.handle_one_request()
    │    └ <function BaseHTTPRequestHandler.handle_one_request at 0x00000265A0CE2160>
    └ <werkzeug.serving.WSGIRequestHandler object at 0x00000265A38C8A60>

  File "D:\Python\Python39\lib\http\server.py", line 413, in handle_one_request
    method()
    └ <bound method WSGIRequestHandler.run_wsgi of <werkzeug.serving.WSGIRequestHandler object at 0x00000265A38C8A60>>

  File "D:\Python\Python39\lib\site-packages\werkzeug\serving.py", line 362, in run_wsgi
    execute(self.server.app)
    │       │    │      └ <werkzeug.debug.DebuggedApplication object at 0x00000265A36D86A0>
    │       │    └ <werkzeug.serving.BaseWSGIServer object at 0x00000265A36D8FD0>
    │       └ <werkzeug.serving.WSGIRequestHandler object at 0x00000265A38C8A60>
    └ <function WSGIRequestHandler.run_wsgi.<locals>.execute at 0x00000265A38B5EE0>

  File "D:\Python\Python39\lib\site-packages\werkzeug\serving.py", line 325, in execute
    for data in application_iter:
                └ <generator object DebuggedApplication.debug_application at 0x00000265A3799430>

  File "D:\Python\Python39\lib\site-packages\werkzeug\debug\__init__.py", line 330, in debug_application
    app_iter = self.app(environ, start_response)
               │    │   │        └ <function WSGIRequestHandler.run_wsgi.<locals>.start_response at 0x00000265A38B58B0>
               │    │   └ {'wsgi.version': (1, 0), 'wsgi.url_scheme': 'http', 'wsgi.input': <_io.BufferedReader name=1144>, 'wsgi.errors': <_io.TextIOW...
               │    └ <Flask 'app'>
               └ <werkzeug.debug.DebuggedApplication object at 0x00000265A36D86A0>

  File "D:\Python\Python39\lib\site-packages\flask\app.py", line 2213, in __call__
    return self.wsgi_app(environ, start_response)
           │    │        │        └ <function WSGIRequestHandler.run_wsgi.<locals>.start_response at 0x00000265A38B58B0>
           │    │        └ {'wsgi.version': (1, 0), 'wsgi.url_scheme': 'http', 'wsgi.input': <_io.BufferedReader name=1144>, 'wsgi.errors': <_io.TextIOW...
           │    └ <function Flask.wsgi_app at 0x00000265A131B790>
           └ <Flask 'app'>

  File "D:\Python\Python39\lib\site-packages\flask\app.py", line 2190, in wsgi_app
    response = self.full_dispatch_request()
               │    └ <function Flask.full_dispatch_request at 0x00000265A1318CA0>
               └ <Flask 'app'>

  File "D:\Python\Python39\lib\site-packages\flask\app.py", line 1484, in full_dispatch_request
    rv = self.dispatch_request()
         │    └ <function Flask.dispatch_request at 0x00000265A1318C10>
         └ <Flask 'app'>

  File "D:\Python\Python39\lib\site-packages\flask\app.py", line 1469, in dispatch_request
    return self.ensure_sync(self.view_functions[rule.endpoint])(**view_args)
           │    │           │    │              │    │            └ {}
           │    │           │    │              │    └ 'orders.queryUser'
           │    │           │    │              └ <Rule '/orders/queryUser' (OPTIONS, GET, HEAD) -> orders.queryUser>
           │    │           │    └ {'static': <function Flask.__init__.<locals>.<lambda> at 0x00000265A1F588B0>, 'apiversion.Apiversion': <function View.as_view...
           │    │           └ <Flask 'app'>
           │    └ <function Flask.ensure_sync at 0x00000265A1318EE0>
           └ <Flask 'app'>

  File "E:\code\backend\api\api_1_0\ordersResource\urls.py", line 27, in queryUser
    return  OrdersOtherResource.queryUser()
            │                   └ <classmethod object at 0x00000265A35E73D0>
            └ <class 'api_1_0.ordersResource.ordersOtherResource.OrdersOtherResource'>

> File "E:\code\backend\api\api_1_0\ordersResource\ordersOtherResource.py", line 68, in queryUser
    kwargs = parser.parse_args()
             │      └ <function RequestParser.parse_args at 0x00000265A31784C0>
             └ <flask_restful.reqparse.RequestParser object at 0x00000265A38C75E0>

  File "D:\Python\Python39\lib\site-packages\flask_restful\reqparse.py", line 328, in parse_args
    value, found = arg.parse(req, self.bundle_errors)
                   │   │     │    │    └ False
                   │   │     │    └ <flask_restful.reqparse.RequestParser object at 0x00000265A38C75E0>
                   │   │     └ <Request 'http://127.0.0.1:5000/orders/queryUser' [GET]>
                   │   └ <function Argument.parse at 0x00000265A3178310>
                   └ Argument('UserID', default=None, dest=None, required=True, ignore=False, location=form, type="<class 'str'>", choices=(), act...

  File "D:\Python\Python39\lib\site-packages\flask_restful\reqparse.py", line 246, in parse
    self.handle_validation_error(ValueError(error_msg), bundle_errors)
    │    │                                  │           └ False
    │    │                                  └ 'Missing required parameter in the post body'
    │    └ <function Argument.handle_validation_error at 0x00000265A3178280>
    └ Argument('UserID', default=None, dest=None, required=True, ignore=False, location=form, type="<class 'str'>", choices=(), act...

  File "D:\Python\Python39\lib\site-packages\flask_restful\reqparse.py", line 173, in handle_validation_error
    flask_restful.abort(400, message=msg)
    │             │                  └ {'UserID': '用户ID数据类型不匹配'}
    │             └ <function abort at 0x00000265A312BD30>
    └ <module 'flask_restful' from 'D:\\Python\\Python39\\lib\\site-packages\\flask_restful\\__init__.py'>

  File "D:\Python\Python39\lib\site-packages\flask_restful\__init__.py", line 32, in abort
    original_flask_abort(http_status_code)
    │                    └ 400
    └ <function abort at 0x00000265A117C670>

  File "D:\Python\Python39\lib\site-packages\flask\helpers.py", line 277, in abort
    current_app.aborter(code, *args, **kwargs)
    │                   │      │       └ {}
    │                   │      └ ()
    │                   └ 400
    └ <Flask 'app'>

  File "D:\Python\Python39\lib\site-packages\werkzeug\exceptions.py", line 861, in __call__
    raise self.mapping[code](*args, **kwargs)
          │    │       │      │       └ {}
          │    │       │      └ ()
          │    │       └ 400
          │    └ {400: <class 'werkzeug.exceptions.BadRequest'>, 401: <class 'werkzeug.exceptions.Unauthorized'>, 403: <class 'werkzeug.except...
          └ <werkzeug.exceptions.Aborter object at 0x00000265A0B63970>

werkzeug.exceptions.BadRequest: 400 Bad Request: The browser (or proxy) sent a request that this server could not understand.
2024-01-07 03:26:19.366 | ERROR    | utils.loggings:exception:40 - 400 Bad Request: The browser (or proxy) sent a request that this server could not understand.
Traceback (most recent call last):

  File "D:\Python\Python39\lib\threading.py", line 937, in _bootstrap
    self._bootstrap_inner()
    │    └ <function Thread._bootstrap_inner at 0x00000265FFC74670>
    └ <Thread(Thread-1, started daemon 30420)>

  File "D:\Python\Python39\lib\threading.py", line 980, in _bootstrap_inner
    self.run()
    │    └ <function Thread.run at 0x00000265FFC743A0>
    └ <Thread(Thread-1, started daemon 30420)>

  File "D:\Python\Python39\lib\threading.py", line 917, in run
    self._target(*self._args, **self._kwargs)
    │    │        │    │        │    └ {}
    │    │        │    │        └ <Thread(Thread-1, started daemon 30420)>
    │    │        │    └ ()
    │    │        └ <Thread(Thread-1, started daemon 30420)>
    │    └ <bound method BaseWSGIServer.serve_forever of <werkzeug.serving.BaseWSGIServer object at 0x00000265A36D8FD0>>
    └ <Thread(Thread-1, started daemon 30420)>

  File "D:\Python\Python39\lib\site-packages\werkzeug\serving.py", line 806, in serve_forever
    super().serve_forever(poll_interval=poll_interval)
                                        └ 0.5

  File "D:\Python\Python39\lib\socketserver.py", line 237, in serve_forever
    self._handle_request_noblock()
    │    └ <function BaseServer._handle_request_noblock at 0x00000265A0BB7AF0>
    └ <werkzeug.serving.BaseWSGIServer object at 0x00000265A36D8FD0>

  File "D:\Python\Python39\lib\socketserver.py", line 316, in _handle_request_noblock
    self.process_request(request, client_address)
    │    │               │        └ ('127.0.0.1', 51380)
    │    │               └ <socket.socket fd=1144, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 5000), raddr=...
    │    └ <function BaseServer.process_request at 0x00000265A0BB7CA0>
    └ <werkzeug.serving.BaseWSGIServer object at 0x00000265A36D8FD0>

  File "D:\Python\Python39\lib\socketserver.py", line 347, in process_request
    self.finish_request(request, client_address)
    │    │              │        └ ('127.0.0.1', 51380)
    │    │              └ <socket.socket fd=1144, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 5000), raddr=...
    │    └ <function BaseServer.finish_request at 0x00000265A0BB7DC0>
    └ <werkzeug.serving.BaseWSGIServer object at 0x00000265A36D8FD0>

  File "D:\Python\Python39\lib\socketserver.py", line 360, in finish_request
    self.RequestHandlerClass(request, client_address, self)
    │    │                   │        │               └ <werkzeug.serving.BaseWSGIServer object at 0x00000265A36D8FD0>
    │    │                   │        └ ('127.0.0.1', 51380)
    │    │                   └ <socket.socket fd=1144, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 5000), raddr=...
    │    └ <class 'werkzeug.serving.WSGIRequestHandler'>
    └ <werkzeug.serving.BaseWSGIServer object at 0x00000265A36D8FD0>

  File "D:\Python\Python39\lib\socketserver.py", line 747, in __init__
    self.handle()
    │    └ <function WSGIRequestHandler.handle at 0x00000265A0DF2550>
    └ <werkzeug.serving.WSGIRequestHandler object at 0x00000265A38C7F10>

  File "D:\Python\Python39\lib\site-packages\werkzeug\serving.py", line 390, in handle
    super().handle()

  File "D:\Python\Python39\lib\http\server.py", line 425, in handle
    self.handle_one_request()
    │    └ <function BaseHTTPRequestHandler.handle_one_request at 0x00000265A0CE2160>
    └ <werkzeug.serving.WSGIRequestHandler object at 0x00000265A38C7F10>

  File "D:\Python\Python39\lib\http\server.py", line 413, in handle_one_request
    method()
    └ <bound method WSGIRequestHandler.run_wsgi of <werkzeug.serving.WSGIRequestHandler object at 0x00000265A38C7F10>>

  File "D:\Python\Python39\lib\site-packages\werkzeug\serving.py", line 362, in run_wsgi
    execute(self.server.app)
    │       │    │      └ <werkzeug.debug.DebuggedApplication object at 0x00000265A36D86A0>
    │       │    └ <werkzeug.serving.BaseWSGIServer object at 0x00000265A36D8FD0>
    │       └ <werkzeug.serving.WSGIRequestHandler object at 0x00000265A38C7F10>
    └ <function WSGIRequestHandler.run_wsgi.<locals>.execute at 0x00000265A38D2160>

  File "D:\Python\Python39\lib\site-packages\werkzeug\serving.py", line 325, in execute
    for data in application_iter:
                └ <generator object DebuggedApplication.debug_application at 0x00000265A3799430>

  File "D:\Python\Python39\lib\site-packages\werkzeug\debug\__init__.py", line 330, in debug_application
    app_iter = self.app(environ, start_response)
               │    │   │        └ <function WSGIRequestHandler.run_wsgi.<locals>.start_response at 0x00000265A38D20D0>
               │    │   └ {'wsgi.version': (1, 0), 'wsgi.url_scheme': 'http', 'wsgi.input': <_io.BufferedReader name=1144>, 'wsgi.errors': <_io.TextIOW...
               │    └ <Flask 'app'>
               └ <werkzeug.debug.DebuggedApplication object at 0x00000265A36D86A0>

  File "D:\Python\Python39\lib\site-packages\flask\app.py", line 2213, in __call__
    return self.wsgi_app(environ, start_response)
           │    │        │        └ <function WSGIRequestHandler.run_wsgi.<locals>.start_response at 0x00000265A38D20D0>
           │    │        └ {'wsgi.version': (1, 0), 'wsgi.url_scheme': 'http', 'wsgi.input': <_io.BufferedReader name=1144>, 'wsgi.errors': <_io.TextIOW...
           │    └ <function Flask.wsgi_app at 0x00000265A131B790>
           └ <Flask 'app'>

  File "D:\Python\Python39\lib\site-packages\flask\app.py", line 2190, in wsgi_app
    response = self.full_dispatch_request()
               │    └ <function Flask.full_dispatch_request at 0x00000265A1318CA0>
               └ <Flask 'app'>

  File "D:\Python\Python39\lib\site-packages\flask\app.py", line 1484, in full_dispatch_request
    rv = self.dispatch_request()
         │    └ <function Flask.dispatch_request at 0x00000265A1318C10>
         └ <Flask 'app'>

  File "D:\Python\Python39\lib\site-packages\flask\app.py", line 1469, in dispatch_request
    return self.ensure_sync(self.view_functions[rule.endpoint])(**view_args)
           │    │           │    │              │    │            └ {}
           │    │           │    │              │    └ 'orders.queryUser'
           │    │           │    │              └ <Rule '/orders/queryUser' (OPTIONS, GET, HEAD) -> orders.queryUser>
           │    │           │    └ {'static': <function Flask.__init__.<locals>.<lambda> at 0x00000265A1F588B0>, 'apiversion.Apiversion': <function View.as_view...
           │    │           └ <Flask 'app'>
           │    └ <function Flask.ensure_sync at 0x00000265A1318EE0>
           └ <Flask 'app'>

  File "E:\code\backend\api\api_1_0\ordersResource\urls.py", line 27, in queryUser
    return  OrdersOtherResource.queryUser()
            │                   └ <classmethod object at 0x00000265A35E73D0>
            └ <class 'api_1_0.ordersResource.ordersOtherResource.OrdersOtherResource'>

> File "E:\code\backend\api\api_1_0\ordersResource\ordersOtherResource.py", line 68, in queryUser
    kwargs = parser.parse_args()
             │      └ <function RequestParser.parse_args at 0x00000265A31784C0>
             └ <flask_restful.reqparse.RequestParser object at 0x00000265A38DA730>

  File "D:\Python\Python39\lib\site-packages\flask_restful\reqparse.py", line 328, in parse_args
    value, found = arg.parse(req, self.bundle_errors)
                   │   │     │    │    └ False
                   │   │     │    └ <flask_restful.reqparse.RequestParser object at 0x00000265A38DA730>
                   │   │     └ <Request 'http://127.0.0.1:5000/orders/queryUser' [GET]>
                   │   └ <function Argument.parse at 0x00000265A3178310>
                   └ Argument('UserID', default=None, dest=None, required=True, ignore=False, location=form, type="<class 'str'>", choices=(), act...

  File "D:\Python\Python39\lib\site-packages\flask_restful\reqparse.py", line 246, in parse
    self.handle_validation_error(ValueError(error_msg), bundle_errors)
    │    │                                  │           └ False
    │    │                                  └ 'Missing required parameter in the post body'
    │    └ <function Argument.handle_validation_error at 0x00000265A3178280>
    └ Argument('UserID', default=None, dest=None, required=True, ignore=False, location=form, type="<class 'str'>", choices=(), act...

  File "D:\Python\Python39\lib\site-packages\flask_restful\reqparse.py", line 173, in handle_validation_error
    flask_restful.abort(400, message=msg)
    │             │                  └ {'UserID': '用户ID数据类型不匹配'}
    │             └ <function abort at 0x00000265A312BD30>
    └ <module 'flask_restful' from 'D:\\Python\\Python39\\lib\\site-packages\\flask_restful\\__init__.py'>

  File "D:\Python\Python39\lib\site-packages\flask_restful\__init__.py", line 32, in abort
    original_flask_abort(http_status_code)
    │                    └ 400
    └ <function abort at 0x00000265A117C670>

  File "D:\Python\Python39\lib\site-packages\flask\helpers.py", line 277, in abort
    current_app.aborter(code, *args, **kwargs)
    │                   │      │       └ {}
    │                   │      └ ()
    │                   └ 400
    └ <Flask 'app'>

  File "D:\Python\Python39\lib\site-packages\werkzeug\exceptions.py", line 861, in __call__
    raise self.mapping[code](*args, **kwargs)
          │    │       │      │       └ {}
          │    │       │      └ ()
          │    │       └ 400
          │    └ {400: <class 'werkzeug.exceptions.BadRequest'>, 401: <class 'werkzeug.exceptions.Unauthorized'>, 403: <class 'werkzeug.except...
          └ <werkzeug.exceptions.Aborter object at 0x00000265A0B63970>

werkzeug.exceptions.BadRequest: 400 Bad Request: The browser (or proxy) sent a request that this server could not understand.
2024-01-07 03:28:44.927 | ERROR    | utils.loggings:exception:40 - 400 Bad Request: The browser (or proxy) sent a request that this server could not understand.
Traceback (most recent call last):

  File "D:\Python\Python39\lib\threading.py", line 937, in _bootstrap
    self._bootstrap_inner()
    │    └ <function Thread._bootstrap_inner at 0x00000265FFC74670>
    └ <Thread(Thread-1, started daemon 30420)>

  File "D:\Python\Python39\lib\threading.py", line 980, in _bootstrap_inner
    self.run()
    │    └ <function Thread.run at 0x00000265FFC743A0>
    └ <Thread(Thread-1, started daemon 30420)>

  File "D:\Python\Python39\lib\threading.py", line 917, in run
    self._target(*self._args, **self._kwargs)
    │    │        │    │        │    └ {}
    │    │        │    │        └ <Thread(Thread-1, started daemon 30420)>
    │    │        │    └ ()
    │    │        └ <Thread(Thread-1, started daemon 30420)>
    │    └ <bound method BaseWSGIServer.serve_forever of <werkzeug.serving.BaseWSGIServer object at 0x00000265A36D8FD0>>
    └ <Thread(Thread-1, started daemon 30420)>

  File "D:\Python\Python39\lib\site-packages\werkzeug\serving.py", line 806, in serve_forever
    super().serve_forever(poll_interval=poll_interval)
                                        └ 0.5

  File "D:\Python\Python39\lib\socketserver.py", line 237, in serve_forever
    self._handle_request_noblock()
    │    └ <function BaseServer._handle_request_noblock at 0x00000265A0BB7AF0>
    └ <werkzeug.serving.BaseWSGIServer object at 0x00000265A36D8FD0>

  File "D:\Python\Python39\lib\socketserver.py", line 316, in _handle_request_noblock
    self.process_request(request, client_address)
    │    │               │        └ ('127.0.0.1', 51504)
    │    │               └ <socket.socket fd=1144, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 5000), raddr=...
    │    └ <function BaseServer.process_request at 0x00000265A0BB7CA0>
    └ <werkzeug.serving.BaseWSGIServer object at 0x00000265A36D8FD0>

  File "D:\Python\Python39\lib\socketserver.py", line 347, in process_request
    self.finish_request(request, client_address)
    │    │              │        └ ('127.0.0.1', 51504)
    │    │              └ <socket.socket fd=1144, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 5000), raddr=...
    │    └ <function BaseServer.finish_request at 0x00000265A0BB7DC0>
    └ <werkzeug.serving.BaseWSGIServer object at 0x00000265A36D8FD0>

  File "D:\Python\Python39\lib\socketserver.py", line 360, in finish_request
    self.RequestHandlerClass(request, client_address, self)
    │    │                   │        │               └ <werkzeug.serving.BaseWSGIServer object at 0x00000265A36D8FD0>
    │    │                   │        └ ('127.0.0.1', 51504)
    │    │                   └ <socket.socket fd=1144, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 5000), raddr=...
    │    └ <class 'werkzeug.serving.WSGIRequestHandler'>
    └ <werkzeug.serving.BaseWSGIServer object at 0x00000265A36D8FD0>

  File "D:\Python\Python39\lib\socketserver.py", line 747, in __init__
    self.handle()
    │    └ <function WSGIRequestHandler.handle at 0x00000265A0DF2550>
    └ <werkzeug.serving.WSGIRequestHandler object at 0x00000265A38DAB20>

  File "D:\Python\Python39\lib\site-packages\werkzeug\serving.py", line 390, in handle
    super().handle()

  File "D:\Python\Python39\lib\http\server.py", line 425, in handle
    self.handle_one_request()
    │    └ <function BaseHTTPRequestHandler.handle_one_request at 0x00000265A0CE2160>
    └ <werkzeug.serving.WSGIRequestHandler object at 0x00000265A38DAB20>

  File "D:\Python\Python39\lib\http\server.py", line 413, in handle_one_request
    method()
    └ <bound method WSGIRequestHandler.run_wsgi of <werkzeug.serving.WSGIRequestHandler object at 0x00000265A38DAB20>>

  File "D:\Python\Python39\lib\site-packages\werkzeug\serving.py", line 362, in run_wsgi
    execute(self.server.app)
    │       │    │      └ <werkzeug.debug.DebuggedApplication object at 0x00000265A36D86A0>
    │       │    └ <werkzeug.serving.BaseWSGIServer object at 0x00000265A36D8FD0>
    │       └ <werkzeug.serving.WSGIRequestHandler object at 0x00000265A38DAB20>
    └ <function WSGIRequestHandler.run_wsgi.<locals>.execute at 0x00000265A38D28B0>

  File "D:\Python\Python39\lib\site-packages\werkzeug\serving.py", line 325, in execute
    for data in application_iter:
                └ <generator object DebuggedApplication.debug_application at 0x00000265A3799430>

  File "D:\Python\Python39\lib\site-packages\werkzeug\debug\__init__.py", line 330, in debug_application
    app_iter = self.app(environ, start_response)
               │    │   │        └ <function WSGIRequestHandler.run_wsgi.<locals>.start_response at 0x00000265A38D24C0>
               │    │   └ {'wsgi.version': (1, 0), 'wsgi.url_scheme': 'http', 'wsgi.input': <_io.BufferedReader name=1144>, 'wsgi.errors': <_io.TextIOW...
               │    └ <Flask 'app'>
               └ <werkzeug.debug.DebuggedApplication object at 0x00000265A36D86A0>

  File "D:\Python\Python39\lib\site-packages\flask\app.py", line 2213, in __call__
    return self.wsgi_app(environ, start_response)
           │    │        │        └ <function WSGIRequestHandler.run_wsgi.<locals>.start_response at 0x00000265A38D24C0>
           │    │        └ {'wsgi.version': (1, 0), 'wsgi.url_scheme': 'http', 'wsgi.input': <_io.BufferedReader name=1144>, 'wsgi.errors': <_io.TextIOW...
           │    └ <function Flask.wsgi_app at 0x00000265A131B790>
           └ <Flask 'app'>

  File "D:\Python\Python39\lib\site-packages\flask\app.py", line 2190, in wsgi_app
    response = self.full_dispatch_request()
               │    └ <function Flask.full_dispatch_request at 0x00000265A1318CA0>
               └ <Flask 'app'>

  File "D:\Python\Python39\lib\site-packages\flask\app.py", line 1484, in full_dispatch_request
    rv = self.dispatch_request()
         │    └ <function Flask.dispatch_request at 0x00000265A1318C10>
         └ <Flask 'app'>

  File "D:\Python\Python39\lib\site-packages\flask\app.py", line 1469, in dispatch_request
    return self.ensure_sync(self.view_functions[rule.endpoint])(**view_args)
           │    │           │    │              │    │            └ {}
           │    │           │    │              │    └ 'orders.queryUser'
           │    │           │    │              └ <Rule '/orders/queryUser' (OPTIONS, GET, HEAD) -> orders.queryUser>
           │    │           │    └ {'static': <function Flask.__init__.<locals>.<lambda> at 0x00000265A1F588B0>, 'apiversion.Apiversion': <function View.as_view...
           │    │           └ <Flask 'app'>
           │    └ <function Flask.ensure_sync at 0x00000265A1318EE0>
           └ <Flask 'app'>

  File "E:\code\backend\api\api_1_0\ordersResource\urls.py", line 27, in queryUser
    return  OrdersOtherResource.queryUser()
            │                   └ <classmethod object at 0x00000265A35E73D0>
            └ <class 'api_1_0.ordersResource.ordersOtherResource.OrdersOtherResource'>

> File "E:\code\backend\api\api_1_0\ordersResource\ordersOtherResource.py", line 68, in queryUser
    kwargs = parser.parse_args()
             │      └ <function RequestParser.parse_args at 0x00000265A31784C0>
             └ <flask_restful.reqparse.RequestParser object at 0x00000265A38E5A90>

  File "D:\Python\Python39\lib\site-packages\flask_restful\reqparse.py", line 328, in parse_args
    value, found = arg.parse(req, self.bundle_errors)
                   │   │     │    │    └ False
                   │   │     │    └ <flask_restful.reqparse.RequestParser object at 0x00000265A38E5A90>
                   │   │     └ <Request 'http://127.0.0.1:5000/orders/queryUser?UserID=0' [GET]>
                   │   └ <function Argument.parse at 0x00000265A3178310>
                   └ Argument('UserID', default=None, dest=None, required=True, ignore=False, location=form, type="<class 'str'>", choices=(), act...

  File "D:\Python\Python39\lib\site-packages\flask_restful\reqparse.py", line 246, in parse
    self.handle_validation_error(ValueError(error_msg), bundle_errors)
    │    │                                  │           └ False
    │    │                                  └ 'Missing required parameter in the post body'
    │    └ <function Argument.handle_validation_error at 0x00000265A3178280>
    └ Argument('UserID', default=None, dest=None, required=True, ignore=False, location=form, type="<class 'str'>", choices=(), act...

  File "D:\Python\Python39\lib\site-packages\flask_restful\reqparse.py", line 173, in handle_validation_error
    flask_restful.abort(400, message=msg)
    │             │                  └ {'UserID': '用户ID数据类型不匹配'}
    │             └ <function abort at 0x00000265A312BD30>
    └ <module 'flask_restful' from 'D:\\Python\\Python39\\lib\\site-packages\\flask_restful\\__init__.py'>

  File "D:\Python\Python39\lib\site-packages\flask_restful\__init__.py", line 32, in abort
    original_flask_abort(http_status_code)
    │                    └ 400
    └ <function abort at 0x00000265A117C670>

  File "D:\Python\Python39\lib\site-packages\flask\helpers.py", line 277, in abort
    current_app.aborter(code, *args, **kwargs)
    │                   │      │       └ {}
    │                   │      └ ()
    │                   └ 400
    └ <Flask 'app'>

  File "D:\Python\Python39\lib\site-packages\werkzeug\exceptions.py", line 861, in __call__
    raise self.mapping[code](*args, **kwargs)
          │    │       │      │       └ {}
          │    │       │      └ ()
          │    │       └ 400
          │    └ {400: <class 'werkzeug.exceptions.BadRequest'>, 401: <class 'werkzeug.exceptions.Unauthorized'>, 403: <class 'werkzeug.except...
          └ <werkzeug.exceptions.Aborter object at 0x00000265A0B63970>

werkzeug.exceptions.BadRequest: 400 Bad Request: The browser (or proxy) sent a request that this server could not understand.
